Undefined symbols for architecture x86_64:
  "_gjs_profiler_add_mark(_GjsProfiler*, long long, long long, char const*, char const*, char const*)", referenced from:
      GjsContextPrivate::set_sweeping(bool) in gjs_context.cpp.o
ld: symbol(s) not found for architecture x86_64

--- gjs/profiler.cpp
+++ gjs/profiler.cpp
@@ -673,8 +673,8 @@
     self->filename = g_strdup(filename);
 }
 
-void _gjs_profiler_add_mark(GjsProfiler* self, gint64 time_nsec,
-                            gint64 duration_nsec, const char* group,
+void _gjs_profiler_add_mark(GjsProfiler* self, int64_t time_nsec,
+                            int64_t duration_nsec, const char* group,
                             const char* name, const char* message) {
     g_return_if_fail(self);
     g_return_if_fail(group);
